### ISSUE TITLE
3D print houses refuse to handle the case.

### ISSUE THREAD

**Issue by GOTO-GOSUB** [Sensirion/smartgadget-sht4x-housing#1]
Not having access to a 3D printer myself, I have submitted the case (and wall mount) designs to a 3D print house to have them make the parts for me.

The wall mount passes their tests without issue, however they have refused to manufacture both of the case parts. I'm not savvy enough to make their required alterations which I will mention here, in the hope that this might help someone else.

> 1, As shown below, for (gadget back), the wall thickness of the indicated red area is not enough which has a high risk of deformation, crack and loss, will this risk be acceptable for you? Shall we proceed as per your files?
> 
> If not, could you please kindly increase it to be at least 1.0mm to proceed? 2.0mm will be much better for the final print.
> 
> Besides, the part has high risks of deformation and warping because of the geometry.
> 
> 
> 2, For (gadget front), several separate shells are detected in your file, sorry to tell you that each file can only support 1 shell (1 model) print.
> 
> The different shells are shown in different color. Please kindly unify all the separate shells into one shell.

![Gadget Rear](https://github.com/user-attachments/assets/4934d528-327f-4580-9134-03962a389460)
![Gadget Front](https://github.com/user-attachments/assets/b501e8df-69f5-4c89-92ef-d738a5d30410)

**Comment by quentin-cy** [comment]
Hello @GOTO-GOSUB 

Apologies for the "slight" delay. 
I could not see the issue in the slicer we use in-house but the CAD software indeed showed 2 bodies.
I now modified it a bit and I believe it should work. Can you check it out with the fab ?
[Housing_Gadget_SHT4x_Front_modified.zip](https://github.com/user-attachments/files/16675335/Housing_Gadget_SHT4x_Front_modified.zip)

Cheers
Q.

**Comment by GOTO-GOSUB** [comment]
Thank you for the updated file - I have just submitted it to my fabricator. I will post back with the outcome.

**Comment by GOTO-GOSUB** [comment]
Hi @qfisch ,

Thanks again for the updated file. That passed the 3D fab's ingest test with a warning about wall thickness, but apart from that making for a slightly wonky fit that will probably straighten itself out over time the case fits OK. Now I've got a few of these in my hands there is another issue. The front of the case that covers the battery is tissue paper thin, so thin in fact that I can read through it and it has almost no tensile strength. Can this please be made bit thicker ? It's probably OK for static gadgets that are out of reach but I doubt it will stand up to repeated handling. Someone - probably me - is going to put their finger through these:


![PXL_20240911_120608770](https://github.com/user-attachments/assets/e75a12b6-f05f-46d1-9a55-aacbf1dbc329)

![PXL_20240911_120521207](https://github.com/user-attachments/assets/7a30e9a6-02b9-47b1-8fc8-9734cfda9011)

![PXL_20240911_120414499](https://github.com/user-attachments/assets/6dbfc7a1-5160-421c-8856-fc33fc5c82e5)

**Comment by quentin-cy** [comment]
Hello @GOTO-GOSUB.

My apologies for the trouble, I do not understand what happened between the testing of the case and the publication here.
I need to investigate a bit in order to find who designed it.
We need to make some checks about that model.
I will come back to you once as soon as I get more info

Sorry again 
Q.

**Comment by GOTO-GOSUB** [comment]
No problem, thank you for looking into it. I will wait on your findings.

**Comment by quentin-cy** [comment]
Hello @GOTO-GOSUB , 
I finally put my hands on the original model and the wall thickness is indeed set to 0.53mm.
Depending on the printer's ability it will indeed give a pretty fragile shell.

I have no time to edit the file unfortunately, but I will post the design files so one can modify them.

Cheers
Q

**Comment by quentin-cy** [comment]
initial issue is solved by [this PR](https://github.com/Sensirion/smartgadget-sht4x-housing/pull/4/)
The thin shell issue is moved to a dedicated one.

**Comment by quentin-cy** [comment]
Hello @GOTO-GOSUB ,
I now made the changes required to have a thicker shell over the battery. 
I doubled it to 1mm, I hope you won't have further issues with it.

Cheers
Q

**Comment by GOTO-GOSUB** [comment]
Thanks, @qfisch - I do not have the skill-set to do that myself. As soon as I have enough work to justify another order with my 3D printing house I will get these remade and update this ticket.

**Comment by GOTO-GOSUB** [comment]
Hi @qfisch ,

I have just had these back from my print house - the changes that you made have resolved this issue.

![Sensirion revised cases (battery)](https://github.com/user-attachments/assets/17167aa4-02e4-4ca3-bdaa-cd6bbaff2d8e)

Thanks again.
